src/tennissimulator/utils/IdGenerator.java

```diff
@@ -0,0 +1,11 @@
+package tennissimulator.utils;
+
+public class IdGenerator {
+
+    public static int playersCounter = 0;
+
+    public static int getId() {
```

Comment:
Maybe name it "generate"?  getId sounds like a simple getter method without any logic. But here you actually increment it.

Reply:
renamed